GSA/digitalgov.gov
# Crowdsourcing and citizen science topic updates

## Summary

Updating content on "Crowdsourcing and citizen science" topic page and updating page to reflect to Topics page format/design

### Preview

https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/ss-crowdsourcing-and-citizen-science-topic-updates/topics/crowdsourcing-and-citizen-science/ 



### Solution
Added new content and updated format of page.

### How To Test

1. Navigate to "Crowdsourcing and citizen science" topic page.
2. Ensure that content on page matches what was approved by Toni and Ammie.
3. Ensure that new format is displaying appropriately and that all resource and community cards have the correct kicker text.

---

### Dev Checklist
